help/bulk_data.md

```diff
@@ -63,3 +63,35 @@ dataset also includes extracted citation data.
 For all articles the [processed PDF and source files are available from
 Amazon S3](bulk_data_s3.md). We recommend this method for bulk access to
 the full-text of arXiv articles.
+
+<span id="harvest"></span>
+Custom Programmatic Harvesting 
+------------------------------
+
+As stated on our [robots](/help/robots) page, arXiv has limited server capacity 
+and our first priority is to support interactive use by human users. That said, 
+we are plainly aware that interested parties will want to make use of our corpus.
+
+### Play nice
+
+We ask that users intent on harvesting use the dedicated pages of `export.arxiv.org` 

```

Comment:
Would it be useful to mention that the export nodes contain a complete up-to-date copy of the corpus so clients are not wondering about what they are getting from 'dedicated pages' on export nodes? 

I'm wondering if something along the lines of 'servers dedicated to automated/bulk downloads' would be more descriptive than 'dedicated pages'.